### ISSUE TITLE
Failure on updating typed_relation field type

### ISSUE THREAD

**Issue by ajstanley** [mjordan/islandora_workbench#358]
If a term doesn't have a url or a uuid, then we see a failure [here](https://github.com/mjordan/islandora_workbench/blob/main/workbench#L596-L598).

This worked for me
```
                for comparison_value in node_comparison_values:
                    if comparison_value.get('url'):
                        del comparison_value['url']
                    if comparison_value.get('target_uuid'):
                        del comparison_value['target_uuid']
```
Let me know if you want a pull request on that.
(I've got some more digging to do - the field is not being updated, but I can get past this chunk of code)

**Comment by mjordan** [comment]
@ajstanley thanks for finding that. If you can open a PR, that would be great.

**Comment by ajstanley** [comment]
Sure thing.  Later this afternoon, I think.

**Comment by mjordan** [comment]
Thanks @ajstanley, resolved with 81491b18083803060e966d6217c2f43e9e9fcc76.
